README.md

```diff
@@ -30,15 +39,9 @@ npm install -g grunt-cli
 npm install
 ```
 
-- Perform an initial build:
```

Comment:
I noticed that `grunt build` no longer exists. Can someone please confirm the updated instructions? 

```
npm i
grunt serve
```